### ISSUE TITLE
Incorrect usage of the term 'API' in project description

### ISSUE THREAD

**Issue by ghost** [taranjeet/chatgpt-api#6]
This project is not an API, but rather a project that utilizes a Chrome-based automation library to open the chat.openai.com website and retrieve the output from that website. This output is not an API, but rather the direct output from the OpenAI website. Therefore, it is not appropriate to refer to this project as an API. Instead, it should be referred to as a project that utilizes the output from the OpenAI website in order to achieve its intended purpose.

**Comment by hugo-alves** [comment]
You must be fun at parties.

**Comment by ghost** [comment]
> You must be fun at parties.

lmao

**Comment by gerrywastaken** [comment]
I think this is actually a joke! It seems like he used ChatGPT to write the issue! The phrasing seems identical and deliberately verbose.

**Comment by ghost** [comment]
> I think this is actually a joke! It seems like he used ChatGPT to write the issue! The phrasing seems identical and deliberately verbose.

😲, damn really?! That's so cool! Thanks for the compliment!

**Comment by ItsAditya-xyz** [comment]
The lord has spoken
![image](https://user-images.githubusercontent.com/55331140/205844876-cf61c873-8078-4c99-b766-fdfacbb21863.png)

**Comment by ghost** [comment]
> The lord has spoken ![image](https://user-images.githubusercontent.com/55331140/205844876-cf61c873-8078-4c99-b766-fdfacbb21863.png)

"It is not accurate to say that using Chromium and Playwright to interact with a program is not an API. In fact, Chromium and Playwright can be used to create and implement APIs.

API stands for application programming interface, and it refers to a set of rules and protocols that allow different software programs to communicate and interact with each other. APIs are a common way for programs to access and exchange data, allowing developers to build new features and functionality on top of existing programs and services.

Chromium is an open-source web browser that is used as the basis for many other popular web browsers, including Google Chrome and Microsoft Edge. Playwright is a tool that allows developers to automate and control Chromium and other web browsers in order to test and develop web applications. Both Chromium and Playwright can be used to create and implement APIs, as well as to interact with existing APIs in various ways.

In conclusion, while it is not accurate to say that using Chromium and Playwright to interact with a program is not an API, it is important to note that both of these tools can be used to create and implement APIs, as well as to interact with existing APIs."

Source: ChatGPT itself
